doc/manualGrading.md

```diff
@@ -0,0 +1,56 @@
+# Manual Grading
+
+PrairieLearn supports manual grading of questions by downloading a CSV file with student answers and uploading a CSV file with question scores and optional per-question feedback. There is not currently an online web interface for manual grading.
+
+
+## Configuring a question for maual grading
+
+The [`info.json` file](question.md#question-infojson) in the question should be set for manual grading:
+```json
+{
+    "uuid": "cbf5cbf2-6458-4f13-a418-aa4d2b1093ff",
+    "gradingMethod": "Manual",
+    "singleVariant": true,
+    ...
+}
+```
+
+Manually-graded questions allow students to "Save" answers, but they don't have a "Save & Grade" button. Instead, the student just saves answers as many times as they want, and all of their submitted answers are stored. It is recommended to also mark manually-graded questions as `"singleVariant": true` so that students are only given a single random variant, even on Homework assessments.
```

Comment:
Emphasize the `"gradingMethod": "Manual"` key?

Reply:
done